src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -993,10 +993,9 @@ static inline int MPIDI_NM_get_local_upids(MPIR_Comm * comm, size_t ** local_upi
     goto fn_exit;
 }
 
-static inline int MPIDI_OFI_upids_to_lupids_general(int size,
-                                                    size_t * remote_upid_size,
-                                                    char *remote_upids,
-                                                    int **remote_lupids, int use_av_table)
+static inline int MPIDI_NM_upids_to_lupids(int size,
+                                           size_t * remote_upid_size,
+                                           char *remote_upids, int **remote_lupids)
```

Comment:
The rename can probably go with the first commit that moves this function to a different file.